### PR TITLE
[1LP][RFR] update network name locator to network_id on SubnetAddView

### PR DESCRIPTION
**Pull request by jadhaj**
Purpose or Intent
=================
The network name locator was changed from cloud_network_id to network_id on SubnetAddView

{{ pytest: cfme/tests/openstack/cloud/test_networks.py::test_create_subnet }}

### CODE REVIEW

File: cfme/networks/views.py [hunk 1]
```diff
@@ -545,7 +545,7 @@ class SubnetAddView(BaseLoggedInPage):
     title = Text('//div[@id="main-content"]//h1')
     network_manager = Select(id='ems_id')
     cloud_tenant = Select(name='cloud_tenant_id')
-    network = Select(name='cloud_network_id')
+    network = Select(name='network_id')
```
Comment (quarckster) [inline cfme/networks/views.py]: **QUESTION**
Does it work for 5.8 as well?

Comment (jadhaj) [reply cfme/networks/views.py]: yes, it was tested on 5.8

Comment (jadhaj) [reply cfme/networks/views.py]: Works for 5.8 and 5.9. locator name was changed on both

**Comment by izapolsk** [comment]
@jadhaj , please test your tests locally. PRT still fails.

**Pull request merged**